test/test_buffer.cpp

```diff
@@ -32,8 +32,12 @@ TEST_CASE("NelemHeuristics","[buffer]") {
   int half=buffer::n_buffer_elem_heuristics(0.5, element_size, nKQ);
   int quarter=buffer::n_buffer_elem_heuristics(0.25, element_size, nKQ);
 
-  REQUIRE_THAT(all, Catch::Matchers::WithinAbs( 2.*half, 2.));
-  REQUIRE_THAT(all, Catch::Matchers::WithinAbs(4.*quarter, 2.));
+  std::cout << "GH says: all = " << all << std::endl;
+  std::cout << "GH says: half = " << half << std::endl;
+  std::cout << "GH says: quarter = " << quarter << std::endl;
```

Comment:
sure you want these outputs here?

Reply:
Forgot to refactor these in - one of the tests here failed on my system, so I was checking what was the issue. I will clean these out.